src/ui/QuerySummary/QuerySummary.ts

```diff
@@ -294,10 +288,24 @@ export class QuerySummary extends Component {
     }
   }
 
+  private hideCustomNoResultsFoundPage() {
+    const showIfNoResultsElement = this.element.getElementsByClassName(SHOW_IF_NO_RESULTS);
```

Comment:
You can do `$$(this.element).find(.`${SHOW_IF_NO_RESULTS}`)`;

Reply:
I will change for this logic no problem!